### ISSUE TITLE
update/set tween progress programatically

### ISSUE THREAD

**Issue by HannesGitH** [thednp/kute.js#113]
i want to set the progress programatically but the `update` method doesn't work like i expected

see example on [codepen](https://codepen.io/HannesGitH/pen/XWqEEzJ)

is this an issue with kute itself or am i doing something wrong?
if the latter please update the docs on how to use the update method / set progress programatically

PS: yes, i've seen the official way with binding to a slider, but in my actual scenario i dont want a slider (yes i could probably build a workaround with a hidden slider whose mouseevents i trigger programatically, but that would be bad)

**Comment by HannesGitH** [comment]
and even better would be setting the progress of combined tweens (commented in the codepen)

**Comment by thednp** [comment]
Yes, because KUTE was developed different from it's parents (tween.js, raphael, etc). We may have to revisit this at some point in time. I'm thinking on ways to make the tween object even smaller, a bit more flexible, and disconnect a DOM node from it, so you should be able to use it with any UI framework there is.

**Comment by thednp** [comment]
Perhaps this will help.

https://thednp.github.io/kute.js/progress.html

**Comment by HannesGitH** [comment]
> Perhaps this will help.
> 
> https://thednp.github.io/kute.js/progress.html

yep this is the one i meant 

> PS: yes, i've seen the official way with binding to a slider, but in my actual scenario i dont want a slider (yes i could probably build a workaround with a hidden slider whose mouseevents i trigger programatically, but that would be bad)

but as you guessed correctly, i want to use it with svelte and have the animationProgress coupled to some state

**Comment by thednp** [comment]
The best I can do right now is to link you the actual [ProgressBar](https://github.com/thednp/kute.js/blob/master/src/util/progressBar.js), perhaps you can find a way to adapt to your Svelte app.

**Comment by HannesGitH** [comment]
thats where im coming from and 
https://github.com/thednp/kute.js/blob/9f8b874567601d038161ee3d1fbac3a692db39bc/src/util/progressBar.js#L78
gave me the idea for my codepen linked above.

but thanks for your help :)
(imma switch to flubber for now tho)

**Comment by brian2524** [comment]
Was hoping this would be possible. Any eta on a feature like this or no?
